### PR TITLE
added support for feed logos and item enclosures

### PR DESCRIPTION
**Pull request by nek0**
I hope I didn't mess things up too badly ...

### CODE REVIEW

File: yesod-newsfeed/Yesod/Common.hs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Yesod.Common
+  ( removeItem
```
Comment (gregwebs) [inline yesod-newsfeed/Yesod/Common.hs]: what is this used for?

**Comment by gregwebs** [comment]
Thanks for the contribution! Can you add an entry to the ChangeLog.md ?

**Comment by nek0** [comment]
sure.

File: yesod-newsfeed/Yesod/Common.hs (hunk 1 above)
Comment (nek0) [inline yesod-newsfeed/Yesod/Common.hs]: it isn't used anymore. I used it to clean the lists up a bit, which didn't work out.

**Comment by nek0** [comment]
changelog has arrived

**Comment by gregwebs** [comment]
Thanks for the great Changelog! Unfortunately this is a breaking change. Let me know if you have any other breaking changes in mind. If not we can release this now with a version bump

**Comment by nek0** [comment]
I have no more changes in mind.

**Comment by gregwebs** [comment]
@snoyberg I need hackage access to upload this package. yesod-newsfeed/1.5 tag is already pushed.

**Comment by snoyberg** [comment]
Done

**Comment by nek0** [comment]
Hackage still ships 1.4.0.1 as most recent version...

**Comment by gregwebs** [comment]
uploaded now

**Pull request merged**